doc/translation_process.md

```diff
@@ -44,11 +44,11 @@ Visit the [Transifex Signup](https://www.transifex.com/signup/) page to create a
 You can find the Bitcoin translation project at [https://www.transifex.com/bitcoin/bitcoin/](https://www.transifex.com/bitcoin/bitcoin/).
 
 ### Installing the Transifex client command-line tool
-The client is used to fetch updated translations. If you are having problems, or need more details, see [https://docs.transifex.com/client/installing-the-client](https://docs.transifex.com/client/installing-the-client)
+The client is used to fetch updated translations. If you are having problems, or need more details, see https://developers.transifex.com/docs/cli.
 
-`pip install transifex-client`
+`curl -o- https://raw.githubusercontent.com/transifex/cli/master/install.sh | bash` (Installing latest varsion with Linux/Mac)
```

Comment:
I agree. The part regarding to the `~/.transifexrc` file is outdated as well and might be dropped. This file is generated automatically after providing a token during the first run:
```
API token not found. Please provide it and it will be saved in '~/.transifexrc'.
If you don't have an API token, you can generate one in https://app.transifex.com/user/settings/api/
> 
```

The autogenerated `~/.transifexrc` file looks like that:
```
$ cat ~/.transifexrc
[https://app.transifex.com]
rest_hostname = https://rest.api.transifex.com
token         = [REDACTED]
```

Reply:
I'd leave the `~/.transifexrc` in case the user creates an account in Transifex directly, instead of using another platform account like GitHub.

Reply:
I've reconsidered @hebasto suggestion and will drop the `~/.transifexrc` section.